jest.config.ts

```diff
@@ -38,38 +38,40 @@ const config: Config = {
     testMatch: ["<rootDir>/test/unit-tests/tchap/**/*-test.[tj]s?(x)"], // :TCHAP: only tchap tests
     setupFiles: ["jest-canvas-mock"],
     setupFilesAfterEnv: [
-        "<rootDir>/node_modules/matrix-react-sdk/test/setupTests.ts",
+        "<rootDir>/linked-dependencies/matrix-react-sdk/test/setupTests.ts",
```

Comment:
This file will conflict in the future  because of the changes in it. 
If it's just this one file (I can't see because of too many changed files) then it's probably acceptable, since it hopefully will not change too much. If there is many more, then I suggest going back to the yarn links to avoid having to make changes.

(I know it's a bit annoying, but minimizing the quantity of changes is really a gift for Marc-of-the-future :) )